resources/assets/sass/home.scss

```diff
@@ -0,0 +1,14 @@
+// Colors "borrowed" from SimAware. Thanks!
+
```

Comment:
As long as this file is included in assets/app.css, it should be included automatically? @kjporter we had a long discussion about this a few months ago - do you remember?

Reply:
It *should* be, but I don't think that we fully resolved this before. I don't recall... @iccowan